nf_conntrack_maxを1000000に設定

ch1aki opened the pull request:
デフォルトのnf_conntrackの上限(31740)に達してパケットがドロップされてしまうため、十分大きな値を設定するようにします。

Commit:
nf_conntrack_maxを1000000に設定

デフォルトのnf_conntrackの上限(31740)に達してパケットがドロップされてしまうため、
十分大きな値を設定する。

Commit:
CUSTOM_ELEMENTSにkernel-modulesを追加

buty4649 commented:
PRありがとう :+1: 
ただ、kernel-modueというelementはなさそうかな？
https://docs.openstack.org/diskimage-builder/latest/elements.html

ちょうど、sysctlというelementがあるのでこれを使うのがよさそうかな。
https://opendev.org/openstack/diskimage-builder/src/tag/2.23.0/diskimage_builder/elements/sysctl

- - - -
ちなみに、elementの使い方はここらへんに書いてあるので参考までに。
https://docs.openstack.org/diskimage-builder/latest/developer/developing_elements.html

ch1aki commented:
なるほど〜雰囲気でやっていました🙏
リンク先みて出直してきます!

Commit:
nf_conntrack_max をsysctl-write-valueで設定する

以下を参考に、nf_conntrack_maxを設定する。

https://github.com/openstack/octavia/blob/c57ae8d3a75461c9d7d0a69cca0219f539539fca/elements/haproxy-octavia/post-install.d/20-haproxy-tune-kernel#L19-L22

buty4649 reviewed (approved):
thx !!!!
